docs/querying/querying.md

```diff
@@ -76,7 +76,7 @@ Druid has numerous query types for various use cases. Queries are composed of va
 
 ## Which query should I use?
 
-Where possible, we recommend using [Timeseries]() and [TopN]() queries instead of [GroupBy](). GroupBy is the most flexible Druid query, but also has the poorest performance.
+Where possible, we recommend using [Timeseries](../querying/timeseriesquery.md) and [TopN](../querying/topnquery.md) queries instead of [GroupBy](../querying/groupbyquery.md). GroupBy is the most flexible Druid query, but also has the poorest performance.
```

Comment:
I think this description is out of date and this whole section should be deleted.

Reply:
@fjy, are you saying that this recommendation is _not_ true anymore?